src/db/mongo.ts

```diff
@@ -1,6 +1,7 @@
 import { MongoClient, Db, Collection, ObjectId } from 'mongodb';
```

Comment:
There is a more up-to-date version of this file -- apparently my dev was behind master on this repo.  Sorry about that but it should be a simple update.